### ISSUE TITLE
Request for video playback

### ISSUE THREAD

**Issue by bigshow104** [Caleb9/syno-photo-frame#15]
Is it possible to include playback of videos on the album? 

I have a video in the shared album but it only shows the thumbnail image.

Loving the app. this is something I have wanted to make the Synology system even better. I have this running on a PI3 b+.

**Comment by Caleb9** [comment]
No, unfortunately it's not currently possible, and I don't see a chance for it to be added anytime soon :(. I am aware of this limitation and I've been experimenting with implementation but it is simply beyond my skills. It most likely requires usage of ffmpeg and I haven't figured a way to read a video from the API using available ffmpeg Rust bindings. ffmpeg in itself is quite a handful, and with the way Synology Photos serves videos would probably require implementing some kind of buffering on RPi...

Either way I gave up on the idea, for now at least. Would be great to get someone who's an expert to add it to the project, so I'll leave this issue open.
